cases/management/commands/enrich_ciaa_related_entities.py

```
@@ -0,0 +1,276 @@
+                        "entity_type": "ORGANIZATION" if rel_type == "location" else "UNKNOWN",
+                        "nes_id": nes_id
+                    }
+                )
+                if not created and not entity.nes_id and nes_id:
+                    entity.nes_id = nes_id
+                    entity.save(update_fields=["nes_id"])
+                    
+                if created:
+                    self.stats["entities_created"] += 1
+
+                rel, rel_created = CaseEntityRelationship.objects.get_or_create(
```

Ruff (0.15.15)

[warning] Unpacked variable `rel` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)